Operator docs for Source.tick (#28225)
akka/akka: akka-docs/src/main/paradox/stream/operators/Source/tick.md

## Description

A periodical repetition of an arbitrary object. Delay of first tick is specified
separately from interval of the following ticks. 

If downstream is applying backpressure when the time period has passed the tick is dropped.

The source materializes a @apidoc[Cancellable] that can be used to complete the source.

@@@note

The element must be immutable as the source can be materialized several times and may pass it between threads, see the second 

> example for achieving a periodical element that changes over time.

Reviewer: A bit of a run on.

@@@

## Examples

This first example prints to standard out periodically:

Scala
:   @@snip [Tick.scala](/akka-docs/src/test/scala/docs/stream/operators/source/Tick.scala) { #simple }

Java
:   @@snip [Tick.java](/akka-docs/src/test/java/jdocs/stream/operators/source/Tick.java) { #simple }

You can also use the tick to periodically emit a value, in this sample we use the tick to trigger a query to an
actor using @ref:[ask](../../../typed/interaction-patterns.md#outside-ask) and emit the response downstream. For this

> usage we don't really care about the actual tick value, only that it was emitted:

Reviewer: ```suggestion
usage, what is important is that it was emitted, not the actual tick value.
```

## Description

A periodical repetition of an arbitrary object. Delay of first tick is specified

> separately from interval of the following ticks. 

Reviewer 2: ```
A source periodically emitting an arbitrary element. The delay of the first tick is specified
separately from the interval of the following ticks. 
```

If downstream is applying backpressure when the time period has passed the tick is dropped.

The source materializes a @apidoc[Cancellable] that can be used to complete the source.

@@@note

> The element must be immutable as the source can be materialized several times and may pass between threads, see second 

Reviewer 2: ```suggestion
The element must be immutable as the source can be materialized several times and may pass it between threads, see the second 
```

example for achieving a periodical element that changes over time.

@@@

> 

Reviewer 2: Possibly add a reference to `throttle`.

## Examples

> Let's start with a simple sample, just printing to standard out periodically:

Reviewer 2: ```suggestion
This first example prints to standard out periodically:
```
